client/src/Components/Mentors.js

```diff
@@ -1,143 +1,138 @@
+		fetch("http://localhost:3100/api/quiz", {
+			method: "POST",
+			headers: { "Content-type": "application/json" },
+			body: JSON.stringify(newQuiz),
+		})
 			.then((response) => response.json())
+			.then((data) => console.log(data))
 			.catch((err) => console.error(err));
 	};
-	const newQuizName =(e)=>{
-
-		setNewQuizz({ ...newQuizz,
-			name:e.target.value,
-		},
-		);
+	const newQuizName = (event) => {
+		setNewQuiz({
+			...newQuiz,
+			publishingDate:dayjs().format(),
+			name: event.target.value,
+		});
 	};
-
-	const removeQuestion =(e)=>{
-		let filterIdArr=newQuizz.questions_id.filter((boo) => {
-			return(boo!= e.target.value);
+	const removeQuestion = (event) => {
+		let filteredQustionIds = newQuiz.questions_id.filter((questionId) => {
+			return questionId != event.target.value;
+		});
+		setNewQuiz({
+			...newQuiz,
+			questions_id: filteredQustionIds,
 		});
-		setNewQuizz({ ...newQuizz,
-			questions_id:filterIdArr,
-		},
-		setRefreshQuestions(!refreshQuestions)
+		setNewQuizQuestions(
+			newQuizQuestions.filter((question) => question._id !== event.target.value)
 		);
-		setNewQuizzQuestions(newQuizzQuestions.filter((obj) => obj._id != e.target.value));
 	};
-	if(props.questions){
-		return(<div className='row'>
-			<div className='col-9 cardBlock'>
-				{props.questions.map((quest)=>
-					<div className='col-3 card'>
-
-						<button id={quest._id} value={quest._id} onClick={addQuestion}>add to quiz</button>
-
-						<div className="quizzQuestion">{quest.question}</div>
-
-						<div className='answers'>
-							{Object.entries(quest.answers).map(([key, value]) =>{
-
-								return(
-									<div>
-										<div className='col-6 answer'>{value}</div>
-									</div>);
-							}
-			  )}</div>
-			  </div>
-				)
-				}</div>
-			<div className='col-3 newQuiz'><h1>New quiz</h1>
-				<input type='text' onKeyUp={newQuizName} placeholder={"new quiz name"} />
-				{newQuizzQuestions.map((quest)=>
-					<div className='col-12 card' key={quest.question}>
- 	<button key={quest._id+quest.question} type="checkbox" checked='checked' id="horns" key={quest._id+quest.question} value={quest._id} onClick={removeQuestion} >x</button>
-						<label htmlFor={quest._id+quest.question}>remove from quizz</label>
-						<div>{quest.question}</div>
-
-						<div className='answers'>
-							{Object.entries(quest.answers).map(([key, value]) =>{
-							// use keyName to get current key's name
-								return(
-									<div>
-
-
-										<div className='col-6'>{value}</div>
-
-									</div>);
-
-							}
-			  )}</div>
-
-
-			  </div>
-				)
-				}<button onClick={submitNewQ}>submit new quizz</button></div>
-
-		</div>
+	if (props.questions) {
+		return (
+			<div className="row">
+				<div className="col-8 cardBlock">
+					{props.questions.map((question, index) => (
+						<div className="col-6 card" key={index}>
+							<div className="quizzQuestion">
+								<strong>{question.question}</strong>
```

Comment:
this one is going to be markdon 1000% no need to make it strong, but if it just looks better, thatțs actually welcome